deploy/manifests/walrus-file.yaml

```diff
@@ -13,6 +13,7 @@ resources:
     attributes:
       containers:
         - image: sealio/casdoor:v1.344.0-seal.1
+          profile: run
```

Comment:
Isn't it the default? Should we fix on template side?

Reply:
It is the default in template, but looks like frontend only displays attributes filled in during resource creation. Can @hibig check if we could display schema default value on UI besides the filled in attributes?